vertx-web-graphql/src/main/java/io/vertx/ext/web/handler/graphql/impl/GraphQLHandlerImpl.java

```diff
@@ -198,6 +207,67 @@ private void handlePostQuery(RoutingContext rc, GraphQLQuery query, String opera
     executeOne(rc, query);
   }
 
+  private void handlePostMultipart(RoutingContext rc, String operationName, Map<String, Object> variables) {
+    GraphQLInput graphQLInput;
+    if (!options.isRequestMultipartEnabled()) {
+      rc.fail(415);
+      return;
+    }
+
+    try {
+      graphQLInput = parseApolloAttributes(rc);
+    } catch (Exception e) {
+      rc.fail(400, e);
+      return;
+    }
+
+    handlePostQuery(rc, (GraphQLQuery) graphQLInput, operationName, variables);
+  }
+
+  private GraphQLInput parseApolloAttributes(RoutingContext rc) {
```

Comment:
Can you copy the relevant part of the spec in the method documentation?

https://github.com/jaydenseric/graphql-multipart-request-spec

Reply:
Can you also change the name as this is not Apollo stuff but rather a GraphQL community effort to standardize uploads?